### PR TITLE
feat(servies and category): added services and category section

### PR DESCRIPTION
**Pull request by ashimvaricon**

### CODE REVIEW

File: frontend/src/pages/Products/slider/index.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  const categories = [
+    "Category 1",
+    "Category 2",
+    "Category 3",
+    "Category 4",
+    "Category 5",
+    "Category 6",
+    "Category 7",
+    "Category 8",
+  ];
```
Comment (adars-kafle) [inline frontend/src/pages/Products/slider/index.tsx]: don't forget to separate the constants!

File: frontend/src/pages/Services/index.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                <Typography variant="h5">Fast Delivery </Typography>
+                <Typography
+                  variant="body2"
+                  sx={{
+                    margin: "auto",
+                    width: "300px",
+                    textAlign: "center",
+                    wordWrap: "break-word",
+                  }}
+                >
```
Comment (adars-kafle) [inline frontend/src/pages/Services/index.tsx]: avoid inline styles, instead create `styles.ts` file and add related styles inside the file as objects, which you can use here in this component as `styles.typograpgy`.

**Review by adars-kafle — CHANGES_REQUESTED** [review]
Export constants and inline styles. Everything else looks good!

**Review by adars-kafle — APPROVED** [review]
LGTM

File: frontend/src/pages/Home/styles.ts [hunk 3]
```diff
@@ -1,6 +1,12 @@
 import { styled } from "@mui/material/styles";
 import Box from "@mui/material/Box";
 
+export const BoxStyled = styled(Box)({
+  height: "50vh",
+  backgroundColor: "grey",
+  marginTop: "70px",
+});
```
Comment (Pradip-Singh-X) [inline frontend/src/pages/Home/styles.ts:8]: use theme colors

**Pull request merged**